<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>262</width>
      <height>641</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_groups">
       <property name="text">
        <string>Группы</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="list_groups">
       <property name="viewMode">
        <enum>QListView::ListMode</enum>
       </property>
       <property name="uniformItemSizes">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="selectionRectVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_createGroup">
       <property name="text">
        <string>Создать группу</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_editGroup">
       <property name="text">
        <string>Редактировать группу</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_deleteGroup">
       <property name="text">
        <string>Удалить группу</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_prompt">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Подсказка&lt;br/&gt;Доступен двойной клик по группе/задаче&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>721</width>
      <height>641</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_groupTitle">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_groupDescription">
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>26</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btn_addTask">
         <property name="text">
          <string>Добавить задачу</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_editTask">
         <property name="text">
          <string>Редактировать задачу</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_deleteTask">
         <property name="text">
          <string>Удалить задачу</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btn_addManyTasks">
       <property name="text">
        <string>Добавить несколько задач</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QCheckBox" name="check_favorite">
         <property name="text">
          <string>В избранном</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="list_tasks"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="btn_copyTasks">
         <property name="text">
          <string>Скопировать задачи</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_copyDates">
         <property name="text">
          <string>Скопировать задачи (только дата и время)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1043</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
